routes/metrics.go

```diff
@@ -0,0 +1,21 @@
+package routes
+
+import (
+	"github.com/go-chi/chi"
+	"github.com/stakwork/sphinx-tribes/auth"
+	"github.com/stakwork/sphinx-tribes/handlers"
+)
+
+func MetricsRoutes() chi.Router {
+	r := chi.NewRouter()
+
+	r.Group(func(r chi.Router) {
+		r.Use(auth.PubKeyContext)
```

Comment:
I'm assuming we'll add the admin middleware in the future?

Reply:
Yes, we will.